tests/installation/select_visible_patterns_from_bottom.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Select only YaST Development pattern
+#    You can pass
+#    PATTERNS=yast-development
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    $self->go_to_patterns();
+    my @patterns = grep($_, split(/,/, get_required_var('PATTERNS')));
+    # specific for yast development at the end of list
+    if (get_var('PATTERNS') =~ /yast-development/) {
```

Comment:
this is not dynamic, it is fixed to that pattern specific and needle, but you got the idea :) thanks for making the change reshaping your initial idea.

Could you please replace it with `$self->select_visible_unselected_patterns([@patterns]);` it should work out the box IMO.

Does it work without the tab?

Reply:
I can't replace the assert_and_click("pattern-yast-development"); with $self->select_visible_unselected_patterns([@patterns]); because the function use a 'assert_and_click("$_-pattern")' while the existing tag is pattern_yast-development, the prefix is different.

new VR: http://openqa.nue.suse.com/tests/9776383#step/select_visible_patterns_from_bottom/10

Reply:
yes, you need to create a new needle for this, we are not using existing needles.

Reply:
Updated code and updated VR. I think it still need create a MR to update cases to make PATTERN=yast development, while how about the scope?  All cases with PATTERNS = default,yast-development ? Or to be easy, just remove 'default' from @patterns in code.

Reply:
"Remove 'default' from @patterns in code" makes sense, as this module is very specific, and "default" is not an actual pattern. I'd say go ahead. LGTM.

Reply:
Updated code to remove 'default' from @patterns and update the VR also. Please review it again, thanks.

Reply:
see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15693/files#r1009059205